src/Student.java

```diff
@@ -0,0 +1,49 @@
+public class Student {
+
+    private String name;
+    private int course;
+    private int grade;
+
+    public Student(String name, int course, int grade) {
+        this.name = name;
+        this.course = course;
+        this.grade = grade;
+    }
+
+    public Student() {
```

Comment:
It's constuctor without parameters -   it's the same as default constuctor so it should be deleted for optimization